localstack-core/localstack/services/apigateway/next_gen/execute_api/handlers/method_response.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from localstack.http import Response
+
+from ..api import RestApiGatewayHandler, RestApiGatewayHandlerChain
+from ..context import RestApiInvocationContext
+
+LOG = logging.getLogger(__name__)
+
+
+class MethodResponseHandler(RestApiGatewayHandler):
+    """
+    This class might not too much, we still need to investigate but from a first look, does not have much impact
+    on the HTTP response
```

Comment:
nit: This comment is a bit confusing 😄

Reply:
Yes, sorry, this was mostly internal 😅 we realized with @cloutierMat that the `MethodResponse` resource in APIGW might just be for "decorative purposes" and used to create SDKs from the REST API, but we're not 100% sure this handler does nothing. I'll update the comment!